.github/workflows/python-package.yml

```diff
@@ -132,6 +132,8 @@ jobs:
         # Some of the examples use Jupyter. We need to keep IPython to
         # below 8.0 as we still run the test suite with Python 3.7.
         pip install "ipython>=7.0,<8.0" jupyter
+        # We need to install sphinx to get correct doc testing
+        pip install sphinx
```

Comment:
Could we just change L140 below to be `pip install .[test,doc]`?

Reply:
I'll try it

Reply:
On the same line it was failing for some reason, have moved it to be on 2 lines.